tests/nnstreamer_filter_ncnn/runTest.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+##
+## SPDX-License-Identifier: LGPL-2.1-only
+##
+## @file runTest.sh
+## @author Kijun Shin <sharaelong@snu.ac.kr>
+## @date Dec 16 2023
+## @brief SSAT Test Cases for NNStreamer
+##
+if [[ "$SSATAPILOADED" != "1" ]]; then
+    SILENT=0
+    INDEPENDENT=1
+    search="ssat-api.sh"
+    source $search
+    printf "${Blue}Independent Mode${NC}"
+fi
+
```

Comment:
Add dependency library check here. Like in https://github.com/nnstreamer/nnstreamer/blob/main/tests/nnstreamer_filter_onnxruntime/runTest.sh#L25